data/transactions/logic/doc.go

```diff
@@ -261,6 +267,12 @@ var opcodeImmediateNotes = map[string]string{
 
 	"vrf_verify": "{uint8 parameters index}",
 	"block":      "{uint8 block field}",
+
+	"proto":      "{uint8 arguments} {uint8 return values}",
+	"frame_dig":  "{int8 frame slot}",
+	"frame_bury": "{int8 frame slot}",
```

Comment:
seems the previous `frame_get/frame_put` are renamed to `frame_dig/frame_bury`?

Reply:
Yeah, I noticed that `frame_get` was exactly like `dig` except based of the frame pointer, so I decided to call it that for consistency.  `bury` doesn't exist right now, but I figured `frame_bury` ought to be the reverse of `frame_dig`. Maybe I will add `bury` as a way to _replace_ something in the stack. It seems useful, but right now we only have `cover` which shifts the stack so nothing is replaced.

Reply:
Well, I would advocate `bury` to be the reverse operation of `dig`, rather than replacing stuffs on the stack, for my thinking on `bury` and `dig` should better not change stack height.

We can always pop off stuffs to achieve replacement, right?